hub/integrations/tests/test_tensorflow.py

```diff
@@ -0,0 +1,48 @@
+from hub.util.check_installation import tensorflow_installed
+import numpy as np
+import pytest
+
+
+requires_tensorflow = pytest.mark.skipif(
+    not tensorflow_installed(), reason="requires tensorflow to be installed"
+)
+
+
+@requires_tensorflow
+def test_pytorch_small_old(local_ds):
+    local_ds.create_tensor("image")
+    local_ds.image.extend(np.array([i * np.ones((300, 300)) for i in range(256)]))
+    local_ds.create_tensor("image2")
+    local_ds.image2.extend(np.array([i * np.ones((100, 100)) for i in range(256)]))
+    local_ds.flush()
+
+    tds = local_ds.tensorflow()
+    for i, batch in enumerate(tds):
+        np.testing.assert_array_equal(batch["image"].numpy(), i * np.ones((300, 300)))
```

Comment:
Hold on, are these batches tensorflow's objects or our own? `.numpy()` isn't super clear if we're computing a ds sample or moving data from a tensor to a numpy array. A comment would be helpful here

Reply:
These are in TensorFlow format, will add a comment.